corehq/pillows/mappings/app_mapping.py

```diff
@@ -49,6 +49,7 @@
                 'copy_history': {'type': 'string'},
                 'copy_of': {'type': 'string'},
                 'cp_is_active': {'type': 'boolean'},
+                'created_from_template': {'type': 'string'},
```

Comment:
@orangejenny FYI
A change in a mapping files requires an accompanying change in the index string at the top for our fab preindex to know to fast reindex it. Fixed here: https://github.com/dimagi/commcare-hq/pull/7885